src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random random = new Random();
+    private static final int COLOR_COUNT = Color.values().length;
```

Comment:
Random is not a constant
```suggestion    
    private static final int COLOR_COUNT = Color.values().length;
    private final Random random = new Random();
```